webapp/channels/src/components/more_channels/more_channels.tsx

```diff
@@ -92,13 +95,24 @@ export default class MoreChannels extends React.PureComponent<Props, State> {
     }
 
     componentDidMount() {
-        this.props.actions.getChannels(this.props.teamId, 0, CHANNELS_CHUNK_SIZE * 2);
+        this.props.actions.getChannels(this.props.teamId, 0, CHANNELS_CHUNK_SIZE * 2).then((result) => {
+            this.props.actions.getChannelsMemberCount(result.data?.map((channel) => channel.id) || []);
```

Comment:
I think it'd be better to not call `getChannelsMemberCount` if `result.data` is missing or empty instead of using `|| []` like we are now

Reply:
Fair point, updated as suggested